### PR TITLE
Make Opacity Effect more obvious

### PR DESCRIPTION
**Pull request by mkrus**
Made the background scene more colourful and less transparent.
Picked colours from the Kuesa logof for the triangles.

Needs to be tested by a non-colour blind person :)

### CODE REVIEW

File: examples/kuesa/car-scene/qml/Wind.qml [hunk 1]
```diff
@@ -29,73 +29,80 @@
...
+        id: t3
         source: "t3.png"
         width: parent.width / 9
         fillMode: Image.PreserveAspectFit
         anchors.bottom: parent.bottom
-        opacity: 0.3
 
         NumberAnimation on x {
-            duration: Math.random() * 6000 + 4000
+            duration: randowDuration()
```
Comment (jjcasmar) [inline examples/kuesa/car-scene/qml/Wind.qml]: s/randowDuration/randomDuration?

**Comment by jjcasmar** [comment]
Works fine, but, to my completely lack of artist taste, doesnt look nice.

File: examples/kuesa/car-scene/qml/Wind.qml (hunk 1 above)
Comment (mkrus) [reply examples/kuesa/car-scene/qml/Wind.qml]: done

**Comment by mkrus** [comment]
removed the colours, restoring the original images, just made them more opaque and made sure the triangles remain in scene most of the time

**Pull request merged**